tests/services/test_storage_routing_service.py

```
@@ -0,0 +1,89 @@
+        loc2_path.mkdir()
+        loc2 = ColdStorageLocation(name="Loc 2", path=str(loc2_path))
+        
+        db_session.add_all([loc1, loc2])
+        db_session.commit()
+        
+        path = monitored_path_factory("Multi Route", "/tmp/hot_multi")
+        path.storage_locations = [loc1, loc2]
+        db_session.commit()
+        
+        # Mock disk usage to favor loc2 (more free space)
+        def mock_usage(p):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `mock_usage`

(ANN202)